cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -194,7 +288,7 @@ func gitIndex(o *indexArgs, runCmd func(*exec.Cmd) error) error {
 		}
 	}
 
-	// create git config with options
+	// create git gitConfiguration with options
```

Comment:
intentional update to comment? Stutters now "git gitConfiguration". Same update below.